.circle/packagecloud.sh

```diff
@@ -37,8 +37,8 @@ function main() {
       fi
       ;;
     *)
-      echo $"Usage: deploy {trusty|xenial|bionic|el6|el7|el8} /tmp/st2-packages"
```

Comment:
Please skip editing the `.circle/packagecloud.sh` file as it's just a generic script and may be used by the other repositories.